modules/twinkleimage.js

```diff
@@ -287,11 +287,11 @@ Twinkle.image.callbacks = {
 			Morebits.wiki.flow.check(talkPageName, function () {
 				var flowpage = new Morebits.wiki.flow(talkPageName, wgULS('通知上传者（', '通知上傳者（') + initialContrib + '）');
 				flowpage.setTopic(wgULS('请补充文件[[:', '請補充檔案[[:') + Morebits.pageNameNorm + wgULS(']]的著作权或来源信息', ']]的著作權或來源資訊'));
-				flowpage.setContent('{{subst:Uploadvionotice|' + Morebits.pageNameNorm + '|flow=yes}}');
+				flowpage.setContent('{{subst:Di-' + params.templatename + '-notice|1=' + mw.config.get('wgTitle') + '}}');
```

Comment:
模板沒有自動補上命名空間，使用wgTitle並不正確，下同。
```suggestion
				flowpage.setContent('{{subst:Di-' + params.templatename + '-notice|1=' + Morebits.pageNameNorm + '}}');
```

Reply:
哦我忘了他没用safesubst....

Reply:
跟有沒有safesubst無關吧？

Reply:
> 跟有沒有safesubst無關吧？

enwiki那边是直接用的mw.config.get('wgTitle')，或者模板有差别？

Reply:
![image](https://user-images.githubusercontent.com/7577245/85536618-d3c31e80-b645-11ea-9086-38de859c9586.png)

Reply:
好吧，模板差別。